packages/components/src/DraftConfigurator/OptionGroup/index.js

```diff
@@ -66,13 +66,19 @@ const OptionGroup = (props) => {
 
   return (
     <>
-      {props.options.map((name) => {
+      {props.options.map((nameOrGroup) => {
```

Comment:
These brackets around `nameOrGroup` are not required here. Not wrong, but this is fine too:

```js
{props.options.map(nameOrGroup => {
```

Reply:
Done